packages/fxa-customs-server/.eslintrc

```diff
@@ -7,6 +7,7 @@
     }
   },
   "rules": {
-    "strict": "off"
+    "strict": "off",
+    "handle-callback-err": "off"
```

Comment:
Instead of adding a bunch of 

```
if (err) {
  console.log(err);
}
```

statements, I opted to ignore the warning for only this service.